tests/plugins/borders.test.ts

```diff
@@ -470,4 +471,14 @@ describe("Grid manipulation", () => {
     expect(getBorder(model, "B3")).toEqual({ bottom: b, top: b, left: b, right: b });
     expect(getBorder(model, "C3")).toEqual({ left: b });
   });
+
+  test("Remove multiple rows before borders at the bottom of the sheet", () => {
+    const b = DEFAULT_BORDER_DESC;
```

Comment:
unfortunately the test passes even without de fix. As I recall from the test I did, the spec and the comment I left here: https://github.com/odoo/o-spreadsheet/pull/4296/files/f5e93842f0a1857ee5feab967354eba8ef768b5e#r1621232954 
the issue arises when we set the borders at the bottom of the sheet.

Make sure that your test is actually fixing the issue ;-) A good way to proceed is to:
- first write the test, make sure it is sound (doing what is should, no less, no more)
- ensure the test fails
- fix the issus
- ensure te test now passes